install/install.go

```diff
@@ -1159,6 +1164,21 @@ func (k *K8sInstaller) Exec(command string, args ...string) ([]byte, error) {
 	return utils.Exec(k, command, args...)
 }
 
+func (k *K8sInstaller) getCiliumVersion() semver.Version {
+	var ersion string
+	if k.params.BaseVersion != "" {
+		ersion = strings.TrimPrefix(k.params.BaseVersion, "v")
+	} else {
+		ersion = strings.TrimPrefix(k.params.Version, "v")
+	}
+	v, err := versioncheck.Version(ersion)
+	if err != nil {
+		k.Log("Unable to parse the provided version %q, assuming it's =1.10.0", k.params.Version)
+		v = versioncheck.MustVersion("1.10.0")
```

Comment:
Why is this hardcoded to 10.0.0? Cannot we fetch the latest stable from `stable.txt`?

Reply:
yeah sure we can. i just kept the original code to limit the scope of this pr: https://github.com/cilium/cilium-cli/blob/dc2ed1ea8fcf91f31149362decee04f3e92b1f24/install/install.go#L1295

Reply:
If this is planned for a follow-up PR, would you mind adding a TODO comment then?

Reply:
i wasn't planning to do it though 😄 i'll add a todo comment anyways

Reply:
done